.ci/pytorch/smoke_test/check_binary_symbols.py

```diff
@@ -53,12 +56,17 @@ def get_symbols(lib: str) -> list[tuple[str, str, str]]:
     return [x.split(" ", 2) for x in lines.decode("latin1").split("\n")[:-1]]
 
 
-def grep_symbols(lib: str, patterns: list[Any]) -> list[str]:
+def grep_symbols(lib: str, patterns: list[Any], symbol_type: str = None) -> list[str]:  # noqa: RUF013
```

Comment:
Why violate lint here?
```suggestion
def grep_symbols(lib: str, patterns: list[Any], symbol_type: str|None = None) -> list[str]:
```